### DISCUSSION TITLE
The "comment section" of Animated Items

### DISCUSSION THREAD

**Discussion by palettemc** [palettemc/animated-items#5] [General]
## Hey there!
This is a **Discussion** page in case anyone has comments to say or questions to ask about the resource pack, without needing to open an issue!

Here is what you **should** do here:
- Ask questions about the resource pack (e.g. "Am I allowed to add this resource pack to my mod?")
- Ask for help on the resource pack (e.g. "How do I load the resource pack?")
- Provide feedback

Here is what you **shouldn't** do here:
- Request to animate for mods / any other Vanilla item - This is done in the Issues tab
- Report bugs - This is also done in the Issues tab

So the point of this Discussion page is for comments or simple questions like "When is the next version coming out?" or "You can improve by doing this..."

Please make sure that you abide by the [GitHub community guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines), and I hope you guys can come up with some ways for me to improve Animated Items!

**Comment by JustinTheBob** [comment]
Hello! Love the animated items resource pack! I was wondering though if you where planning on updating it to work with the new data pack version of the tool trims mod. Thank you!

**Reply by palettemc** [reply]
I have tested the latest version of Tool Trims with Animated Items, and it all seems to work fine for me. Are you sure there is a version that doesn't work with Animated Items?

**Reply by JustinTheBob** [reply]
The Tool Trims mod resource pack is on for these screenshots:
![ToolTrims](https://github.com/user-attachments/assets/7bb6843f-e42c-4e9d-9e10-8abaa8f27d0d)
![ToolTrimsSmithing](https://github.com/user-attachments/assets/8dd838ad-8181-46d2-a3ff-55cd405ba313)
![ToolTrimsSmithingGUI](https://github.com/user-attachments/assets/5e9866c0-49b9-4bd8-bb48-8abff3f78bb6)

Animated Items resource pack is on for these screenshots:
![AnimatedItems](https://github.com/user-attachments/assets/a285b063-1e72-4030-a78d-0234701a06fb)
![AnimatedItemsSmithing](https://github.com/user-attachments/assets/23803c2c-7df9-4282-8429-c6a30b33c32c)
![AnimatedItemsSmithingGUI](https://github.com/user-attachments/assets/67b72fc7-cad0-40a6-a453-1513429d3cbe)
I could have something wrong, idk. This works with the data pack version of the mod right?

**Reply by palettemc** [reply]
If you want the textures to work normally, you'll have to turn on both the Tool Trims resource pack and the Animated Items resource pack, but the Animated Items resource pack has to be above the Tool Trims resource pack. Here is an example:
<img width="583" alt="Screenshot 2024-12-05 at 8 04 30 AM" src="https://github.com/user-attachments/assets/98b6a9ff-a1cb-45bc-9180-e6f1184890da">

**Reply by JustinTheBob** [reply]
Ohhhh okay im stupid, very obvious....thank you! LOVE the pack!

**Reply by palettemc** [reply]
Lol thanks. Also, you aren't stupid :)

**Comment by xShadoWolfx** [comment]
Hello! I was wondering if you would make "Animated Items" compatible with "Axolotl Bucket Variants"?

**Reply by palettemc** [reply]
Hello! I can ask the creator of the resource pack if I can add animations for the different textures. I am a little busy, but I am going to release another version of the resource pack soon. I will let you know if I get permission from the creator of Axolotl Bucket Variants.

**Reply by xShadoWolfx** [reply]
Thx for the answer even tho you are busy! And have a wonderful day! I hope u get the permission to make the two resource pack compatible. I can't wait to see how wonderful the resource pack will look in the future!

**Reply by palettemc** [reply]
Sorry for the late response. Unfortunately, I haven't gotten a response from the owner of the Axolotl Bucket Variants resource pack. This means that I cannot animate their textures. If I do ever get a response after some time, I will add the animations to Animated Items.

**Comment by spectre81** [comment]
HI love the pack
I'm using fabric 1.21.10 and it says the rp is broken or incopatible
Still works I'm just wondering why it gives error

**Reply by palettemc** [reply]
Hello. If the resource pack is considered "broken," but it works perfectly fine, then that is good. But I'm curious about this issue, so which version of the resource pack are you using?

**Comment by altyes11111** [comment]
Hi, really like the texture. 
I'm on fabric 1.21.11
How can I disable only the animated torches (both normal and soul torches)?

**Reply by palettemc** [reply]
Hello, and thank you.
To disable the animated torches, you will need to access the following folder that is inside the resource pack's folder:

assets/minecraft/textures/item

In this folder, delete the following files:

torch_e.png
torch_e.png.mcmeta
torch.png
torch.png.mcmeta
soul_torch_e.png
soul_torch_e.png.mcmeta
soul_torch.png
soul_torch.png.mcmeta

After deleting these files, both the torch and soul torch animations will be disabled.

**Reply by altyes11111** [reply]
After deleting the above, the torch became a purple and black grid block, so I try to delete the torch.json and soul_torch.json in assets/minecraft/models/item.

And it actually works. Just wonder if there's any issue in deleting those json files? Will it cause something unexpected? Should I also delete the entire models folder?

**Reply by palettemc** [reply]
My mistake. I forgot to mention to delete the JSON files as well. All they do is determine which texture to use for the torch item, and since the texture was deleted, the game uses the black and purple grid. So, deleting those files has no consequence and will make the game use the default texture instead.

There is no need to delete the whole models folder, but it will not have any effect on the resource pack or functionality because it is empty now.

**Comment by kinamiexe** [comment]
hi,i have a heavy modded pack and animated item works perfectly fine for some items, but for some other it breaks in a way that it only plays the animation for one sec and stops at a random time and waits a bit here is a quick video showcasing it, im on 1.20.1 which is a cursed version for CIT. CIT Resewn (with connector) doesnt work at all, CIT Reforged results in broken icons so im using Forge CIT (which is someones forge port of Resewn) which works perfectly fine for other items (dont know about animated, only have your pack for that)

https://imgur.com/a/9kJbkAx

**Reply by palettemc** [reply]
Hello, to help you, could you please send a list of all mods in use? I'm assuming that the issue is caused by your mods rather than the resource pack itself. I believe this may be an optimization mod that might be pausing texture animations when they are not rendered, which causes animations to stop from time to time, even when they are rendered.

**Comment by Maxrobloxian** [comment]
Hi, I am doing a modpack and I modified the resourcepack so the torches are not animated, so I'm wondering if I can include the modified version in my modpack.

**Reply by palettemc** [reply]
Hello. Yes, you are allowed to include the modified version of my resource pack in your modpack as long as I am credited. If your modpack is going to be personal, meaning that you will **not** publish it, you do not need to credit me anywhere since it's only for you. But if you publish the modpack, please include a link to my resource pack.
